lib/checkPullRequestReview.js

```diff
@@ -255,6 +277,7 @@ const handlePullRequestReview = async (context) => {
   /**
    * @type {import('probot').Octokit.PullsGetReviewResponse} review
    */
+  console.log('At least reached handlePullRequestReview');
```

Comment:
Remove this.

Reply:
Correcting it in the latest commit Sir.Sorry forgot to remove it. Thanks.

Reply:
Still hasn't been removed.

Reply:
Removed them in the later commits. Sorry for Late Reply.